### PR TITLE
Fixed failing unit tests

### PR DESCRIPTION
**Pull request by copejon**
Updated stale unit tests.

### CODE REVIEW

File: pkg/provisioner/resourcehandlers_test.go [hunk 1]
```diff
@@ -37,11 +37,22 @@ func TestNewCredentialsSecret(t *testing.T) {
 		authSecret   = "test-auth-secret"
 	)
 
+	var T = true
```
Comment (jeffvance) [inline pkg/provisioner/resourcehandlers_test.go]: isn't upper case `T` used to represent a generic type?

File: pkg/provisioner/resourcehandlers_test.go [hunk 2]
```diff
@@ -137,17 +148,31 @@ func TestNewCredentialsSecret(t *testing.T) {
...
 
 	const (
+		obcName   = "test-obc"
 		host      = "http://www.test.com"
 		name      = "bucket-name"
 		port      = 11111
 		region    = "region"
 		subRegion = "sub-region"
 	)
+	var T = true
```
Comment (jeffvance) [inline pkg/provisioner/resourcehandlers_test.go]: ditto

File: pkg/provisioner/resourcehandlers_test.go (hunk 1 above)
Comment (copejon) [reply pkg/provisioner/resourcehandlers_test.go]: ah, right. will fix

**Comment by jeffvance** [comment]
LGTM

**Pull request merged**